charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -8,8 +8,8 @@ sources:
   - https://github.com/newrelic/nri-kubernetes/tree/master/charts/newrelic-infrastructure
   - https://github.com/newrelic/infrastructure-agent/
 
-version: 3.5.5
```

Comment:
Can you generate as well new docs?

I usually do it running:
```
 docker run --rm --volume "$(pwd):/helm-docs" -u $(id -u) jnorwood/helm-docs:latest 
```

Reply:
Sure!